Make.defaults

```diff
@@ -23,7 +23,7 @@ EFI_ARCHES ?= aa64 ia32 x64
 enabled = $(if $(filter undefined,$(origin $(1))),$(3),$(2))
 
 HOSTARCH   = $(shell uname -m | sed s,i[3456789]86,ia32,)
-ARCH	   := $(shell uname -m | sed s,i[3456789]86,ia32,)
+ARCH	   := $(shell uname -m | sed s,i[3456789]86,ia32, | sed s,arm.*,arm,)
```

Comment:
Any reason this isn't one sed invocation with two pairs of `-e REGEXP`  arguments?

Reply:
Just lazy, I guess. Have a new version...